pkg/resources/row_access_policy_acceptance_test.go

```diff
@@ -40,20 +46,25 @@ func TestAcc_RowAccessPolicy(t *testing.T) {
 		CheckDestroy: acc.CheckDestroy(t, resources.RowAccessPolicy),
 		Steps: []resource.TestStep{
 			{
-				ConfigDirectory: config.TestStepDirectory(),
-				ConfigVariables: m(),
+				ConfigDirectory: acc.ConfigurationDirectory("TestAcc_RowAccessPolicy/complete"),
+				ConfigVariables: tfconfig.ConfigVariablesFromModel(t, policyModel),
 				Check: assert.AssertThat(t, resourceassert.RowAccessPolicyResource(t, resourceName).
 					HasNameString(id.Name()).
 					HasDatabaseString(id.DatabaseName()).
 					HasSchemaString(id.SchemaName()).
 					HasFullyQualifiedNameString(id.FullyQualifiedName()).
 					HasCommentString("Terraform acceptance test").
-					HasBodyString("case when current_role() in ('ANALYST') then true else false end"),
-					assert.Check(resource.TestCheckResourceAttr(resourceName, "argument.#", "2")),
-					assert.Check(resource.TestCheckResourceAttr(resourceName, "argument.0.name", "N")),
-					assert.Check(resource.TestCheckResourceAttr(resourceName, "argument.0.type", "VARCHAR")),
-					assert.Check(resource.TestCheckResourceAttr(resourceName, "argument.1.name", "V")),
-					assert.Check(resource.TestCheckResourceAttr(resourceName, "argument.1.type", "VARCHAR")),
+					HasBodyString(body).
+					HasArguments([]sdk.RowAccessPolicyArgument{
```

Comment:
you can create a variable and reuse it in this test

Reply:
Done.